prow/prowjobs/GoogleCloudPlatform/testgrid/testgrid-images.yaml

```diff
@@ -14,3 +14,20 @@ presubmits:
         args:
         - test
         - //...
+
+postsubmits:
+  GoogleCloudPlatform/testgrid:
+  - name: push-testgrid-images
+    cluster: testgrid
+    branches:
+    - master
+    decorate: true
+    run_if_changed: '^cmd/.*$'
+    spec:
+      containers:
+      - image: launcher.gcr.io/google/bazel:0.29.1
+        command:
+        - bazel
+        args:
+        - run
+        - //images:push
```

Comment:
I would expect this to need credentials

Reply:
I suspect you'll need to activate the service account credentials too? So might be best to use something like bazel.sh -- and have this activate the service account if the env var is set.

https://github.com/kubernetes/test-infra/blob/master/hack/bazel.sh

https://github.com/kubernetes/test-infra/blob/3dfd16c0c1574a52764747a394e8ec2594796239/boskos/update_prow_config.sh#L42-L44